misc-tools/dj_make_chroot.in

```diff
@@ -80,6 +82,10 @@ Available architectures:
 Debian: $DEBIAN_ARCHLIST
 Ubuntu: $UBUNTU_ARCHLIST
 
+Environment Overrides:
+DEBPROXY: Sets http(s) proxy to be used for package installation
+DEBMIRROR: Sets mirror URL to be used for package registry
```

Comment:
Would be good to mention that these environment variables are overridden by the command line options.